boards/hail/rust-toolchain.toml

```diff
@@ -3,6 +3,6 @@
 # Copyright Tock Contributors 2023.
 
 [toolchain]
-channel = "stable"
+channel = "1.88.0"
```

Comment:
We talked about this before, and this change seems problematic as it will force users to use `1.88.0` even if newer toolchains are available.

We should instead increment the MSRV in the `Cargo.toml`, such that it'll generate a warning when the toolchain is outdated (where, IIRC, the warning text is bad and @bradjc wasn't happy with that, but it was the best we could do).

Reply:
Done